WordPress/wporg-developer
# Dashicons: Make the page work.

Fixes: #187 

The dashicon page is technically in maintenance mode but still needs to work. The switch to a block theme causes issues when loading `page-dashicons.php`. Therefore we need to import the content into a block template. This PR chooses the quickest route possible and moves all the content into a shortcode.

## Changes
 - Move page-dashicons.php content into shortcode
 - Create page-dashicons block template
 - Delete `page-dashicons.php`
 - Delete `page-dashicons.scss`
    - We have a static .css file. We can just add hotfixes there.
 - Adds a notice banner indicating that icons are no longer being added
    - This is already embedded in the page content but we should call it out. 


## Screenshot
| Desktop | Mobile |
| --- | --- |
| ![localhost_8888_dashicons_ (1)](https://user-images.githubusercontent.com/1657336/217399723-31e33501-75d9-4c06-bd74-2350a7021c87.png) |  ![localhost_8888_dashicons_(iPhone SE)](https://user-images.githubusercontent.com/1657336/217399718-aabc1232-6798-470c-b66f-bf564b347d30.png) |

## How To Test
1. Add content to page (Provided Below)
2. Visit `http://localhost:8888/resources/dashicons/` (it will redirect)
3. Try copying the "random" dashicons html/css/glyph
4. Try filtering.
5. Click on an icon, and expect to see the "header" icon update.
6. Reload the page, and expect to see the icon you previously clicked as the "header" icon.


### Page Content
```

<p>Dashicons is the official icon font of the WordPress admin as of 3.8.</p>



<p>The Dashicons project is no longer accepting icon requests. Here's why:&nbsp;<a href="https://make.wordpress.org/design/2020/04/20/next-steps-for-dashicons/">Next steps for Dashicons</a>.</p>



<p>For any issues that appear within WordPress core, please&nbsp;<a href="https://core.trac.wordpress.org/newticket">create a new ticket on trac</a>. Use the "administration" component and the "UI" focus when creating the new ticket, and be sure to include "Dashicons" somewhere in the text of the ticket.</p>



<p>Dashicons is licensed under&nbsp;<a href="http://www.gnu.org/licenses/gpl-2.0.html">GPLv2</a>, or any later version with font exception.</p>

```
